test/src/unit-class_parser.cpp

```diff
@@ -1620,7 +1620,7 @@ TEST_CASE("parser class")
 
         SECTION("from std::array")
         {
-            std::array<uint8_t, 5> v { {'t', 'r', 'u', 'e'} };
+            std::array<uint8_t, 4> v { {'t', 'r', 'u', 'e'} };
```

Comment:
Could you please add another test case for `std::array<uint8_t, 5> v { {'t', 'r', 'u', 'e', 0} }`?